docs/reference/search/search-your-data/search-using-query-rules.asciidoc

```diff
@@ -0,0 +1,150 @@
+[[search-using-query-rules]]
+== Searching with query rules
+++++
+<titleabbrev>Searching with query rules</titleabbrev>
+++++
+
+[[query-rules]]
+
+preview::[]
+
+_Query rules_ are rules that are defined inside {es} that allow customization of search results for queries that match specified criteria metadata.
+This allows for more control over promoted results, ensuring that specified documents always match defined criteria and are returned at the top of the result list.
+Metadata is defined in the query rule, and is matched against the query criteria.
+Metadata is a string key, and can be anything from a user-entered query string, personalized metadata about users (e.g. country, language, etc). or anything else defined in the rules.
```

Comment:
I find metadata definition a bit confusing here. I was thinking on how to rephrase this:

```suggestion
Query rules use metadata to match a query.
Metadata is provided as part of the `rule_query` as an object. 
It can be anything from a user-entered query string, personalized metadata about users (e.g. country, language, etc).

Query rules define a metadata key that will be used to match the metadata provided in the `rule_query` with the criteria specified in the rule.

When a query rule match the `rule_query` metadata according to its defined criteria, the query rule action is applied to the underlying `organic_query`
```

... maybe adding some small examples at the end of each paragraph will make sense as well to guide the user step by step on what query rules are, what metadata is, and how they match.

Reply:
Good call out, I can leave the personalization stuff for a blog post 👍

Reply:
OK, I've incorporated your suggestions, LMK what you think.